docs/articles/nunit/technical-notes/nunit-internals/specs/Dynamic-Test-Cases-Spec.md

```diff
@@ -3,7 +3,7 @@
 > [!WARNING]
 > This is a draft. The contents may be out of date.
 
-This specification describes changes in how test cases will be generated in NUNit 3.0
+This specification describes changes in how test cases will be generated in NUnit 3.0
```

Comment:
It would be great if this could be eventually gotten rid of as a "spec" with the parts that are implemented included somewhere in the docs.

Reply:
I've tried to incorporate some the the engine-relevant ones in this PR. This one is framework of course, but I just corrected the obvious typo when I spotted it along the way. I'm pretty sure we have an issue tracking this somewhere...